src/protocols/bdx/BdxMessages.cpp

```diff
@@ -166,6 +166,32 @@ size_t TransferInit::MessageSize() const
     return WriteToBuffer(emptyBuf).Needed();
 }
 
+void TransferInit::LogMessage() const
```

Comment:
> Is this to support testing?

Yes

Reply:
This approach to logging is not scalable:

- Having access to logs on production devices is not necessarily guaranteed.
- If changes arise to these logs, it may break test verification steps that rely on such logs, if they are even available to the test harness.

Reply:
Spoke offline. Will add a new `ChipLogAutomation` macro, specifically to output logs needed for test validation. Long term solution is probably to have the TH deal with parsing the BDX packets rather than relying on DUT to log messages.